tools/crm_resource.c

```diff
@@ -1585,13 +1585,13 @@ main(int argc, char **argv)
 
         msg = g_strjoinv("", strv);
         g_set_error(&error, PCMK__EXITC_ERROR, CRM_EX_USAGE, "%s", msg);
+        g_free(msg);
 
         for(i = 0; i < len; i++) {
             free(strv[i]);
         }
+        free(strv);
```

Comment:
Huh, this is not what I would have expected given that strv is a `gchar **`, but ACK if that's what static analysis thinks needs to be done.

Reply:
gchar == char; what matters is how they were allocated, and in this case strv was allocated with calloc(). Perhaps strv should be char** or alternatively allocated with glib functions, but this should work fine